020-Connect/010-Postgres/001-direct-access.mdx

```diff
@@ -0,0 +1,283 @@
+
+You can check the Postgres version with the following command:
+
+```bash
+psql "postgresql://[Workspace Id]:[API key]@[Region].sql.xata.sh:[Port]/[Database]:[Branch]" -c "select version()"
+```
+
+```sql
+                                                   version
+-------------------------------------------------------------------------------------------------------------
+ PostgreSQL 15.4 on aarch64-unknown-linux-gnu, compiled by aarch64-unknown-linux-gnu-gcc (GCC) 9.5.0, 64-bit
+(1 row)
+```
+
+## Extensions
+
+In a shared cluster, no extension except the mandatory [plpgsql](https://www.postgresql.org/docs/current/plpgsql.html) is installed and installing others is forbidden.
+If you need extensions, you need to deploy a [dedicated cluster](/docs/dedicated-cluster#Extensions).
+
+## Export
+
+In order to export a branch with `pg_dump`, you need to specify the connection parameters as indicated in the [connection string](#connection-string) section.
+
+```bash
+pg_dump --no-acl --no-owner --no-comments --no-table-access-method --host=[Region].sql.xata.sh --username=[Workspace Id] --dbname=[Database]:[Branch] --password
+```
+
+The prompt will ask you for the password, which is your API key.
+
+The flags, `--no-acl`, `--no-owner`, `--no-comments` and `--no-table-access-method` are not mandatory, but remove unnecessary noise in the file, creating a cleaner output.
+
+Please refer to the pg_dump [documentation](https://www.postgresql.org/docs/current/app-pgdump.html) for a full list of features.
+
+## Import
+
+The import path depends on your data format, if it's a compressed archive (e.g. `.tar`), you need to use [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html).
+pg_dump offers the flag `--format=tar` to immediately output to tar file format.
+
+```bash
+pg_restore --host=[Region].sql.xata.sh --username=[Workspace Id] --dbname=[Database]:[Branch] --password dumpdata.tar
+```
+
+The parameters are equally organized as for pg_dump.
+
+If the file is a plain SQL file (`.sql`), you can convenitently use psql to immediately import the database branch.
+In the following example, we want to import the file `my_branch.sql` into the database `playground` on the branch `new_branch`
+
+```bash
+psql "postgresql://[Workspace Id]:[API key]@[Region].sql.staging-xata.dev/playground:new_branch" < my_branch.sql
+```
+
+The following limitations apply:
+
+- Some warnings are to be expected during the restore operation due to restrictions from [supported statements](/docs/postgres#supported-statements).
+- The set of supported statements differs between Shared and Dedicated Clusters so the warnings when restoring from the same file may differ depending on the target environment.
+- Managing roles and table ownership is not supported, errors from such statements are expected.
+- Schemas are not supported. Data content and objects are restored under the virtual public schema of the target branch.
```

Comment:
I think telling the user that the public schema they see is "virtual" not necessary.

```suggestion
- Schemas are not supported. Data content and objects are restored under the target branch.
```